### PR TITLE
2457  Update Code Climate ID

### PR DESCRIPTION
**Pull request by rizzkhan1**
### Context
Remove CC reporter id from the yaml file and move to the pipeline as a variable. 
Regenerate a new reporter id.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ variables:
   imageNameBgFindSync: 'manage-courses-bg-find-sync'
   # Support multiple docker-compose files to allow travis and az pipelines to co exist: https://docs.docker.com/compose/extends/#multiple-compose-files
   dockerOverride: 'docker-compose -f docker-compose.yml -f docker-compose.az.yml'
-  cc_Reporter_ID: 'de006d3ac97520a939b8c9b67af12b9e85438b9e9fd4255e78651800b36f48ae'
-  AGENT_JOBSTATUS: $(Agent.JobStatus)
```
Comment (dankmitchell) [inline azure-pipelines.yml]: Did you mean to remove this line too?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -12,8 +12,8 @@ variables:
...
+  cc_Reporter_ID: '$(ccReporterID)'
```
Comment (dankmitchell) [inline azure-pipelines.yml]: Why is this snake case?

File: azure-pipelines.yml (hunk 1 above)
Comment (rizzkhan1) [reply azure-pipelines.yml]: yes

Comment (dankmitchell) [reply azure-pipelines.yml]: Okay

File: azure-pipelines.yml (hunk 2 above)
Comment (rizzkhan1) [reply azure-pipelines.yml]: removed snake case variable

**Pull request merged**